blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientBuilder.scala

```diff
@@ -307,6 +333,20 @@ object BlazeClientBuilder {
       channelOptions = ChannelOptions(Vector.empty)
     ) {}
 
+  /** Creates a BlazeClientBuilder
+    *
+    * @param executionContext the ExecutionContext for blaze's internal Futures
+    * @param sslContext Some `SSLContext.getDefault()`, or `None` on systems where the default is unavailable
+    */
+  @deprecated(message = "Use BlazeClientBuilder#apply(ExecutionContext).", since = "1.0.0")
```

Comment:
TODO - look at Ross's answer to https://github.com/http4s/http4s/pull/3422/files#r426109792 to confirm this version is accurate.

Reply:
This will probably appear in 1.0.0-M1.  If not, we'll clean it up at release time.